services/frontend/www-app/quasar.config.js

```diff
@@ -90,8 +90,6 @@ module.exports = configure(function (/* ctx */) {
           .use('i18n')
           .loader('@intlify/vue-i18n-loader');
       },
-
-      vitePlugins: [['vite-plugin-rewrite-all']],
```

Comment:
FYI @ellenhp 

See https://github.com/headwaymaps/headway/pull/281/commits/8db220d69291a0e7fdef12bb274639000f4d6475 - it broke after running `yarn upgrade`.

I didn't look into it very much, but I'm always happy to "fix things" by deletion... LMK if I've missed a use case.

Reply:
I can't remember what I used this for but it must have been something to do with like, url rewriting? I think that's coming up on a year ago and it's not documented so it must not be important. Happy birthday to this project!